packages/expo-doctor/src/utils/log.ts

```diff
@@ -0,0 +1,257 @@
+import chalk from 'chalk';
```

Comment:
The new [expo cli log file](https://github.com/expo/expo/blob/main/packages/%40expo/cli/src/log.ts) is better if you're copy/pasting. You may want to add [something like this too](https://github.com/expo/expo/blob/fd7776f7c9b00648bc3e75580a925d32bf71b1fe/packages/%40expo/cli/bin/cli.ts#L7-L12), to support EXPO_DEBUG and DEBUG.